terraform/modules/rds/main.tf

```diff
@@ -18,8 +18,8 @@ resource "aws_db_instance" "droits" {
   engine                 = "postgres"
   engine_version         = "14"
   instance_class         = "db.t3.micro"
-  username               = var.db_username
-  password               = var.db_password
+  username               = local.envs["DB_USERNAME"]
```

Comment:
so instead of using local.envs, I can set them locally in Bash by exporting them as e.g TF_VAR_db_username and access the sensitive values using var.db_name?

Reply:
Yep, so could just have a local file 


          export TF_VAR_ecr_repository_url = xxx
          export TF_VAR_api_backoffice_ecr_repository_name = xxx
          export TF_VAR_webapp_ecr_repository_name = xxx
          export TF_VAR_backoffice_image_tag = xxx
          export TF_VAR_aws_account_number = xxx
          export TF_VAR_db_username = xxx
          export TF_VAR_db_password = xxx